modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -856,6 +882,56 @@ private String _getLocation(String name, long primaryKey) {
 			_getLocation(name), StringPool.SLASH, primaryKey);
 	}
 
+	private List<ObjectFieldSetting> _getObjectFieldSettings(
```

Comment:
To simplify, move this method to the Util class, then you don't have to worry about the ObjectFieldSettings list (you or whoever uses the Util class)

Reply:
Make sense @ccorreagg!!